README.md

```diff
@@ -85,10 +85,6 @@ If you have questions about working with Solid or just want to share what you’
 - For public feedback, bug reports, and feature requests please file an issue via [GitHub](https://github.com/inrupt/solid-client-authn/issues/).
 - For non-public feedback or support inquiries please use the [Inrupt Service Desk](https://inrupt.atlassian.net/servicedesk).
 
-## Prerequisite
-
-The `solid-client-authn` libraries are compatible with [NSS](https://github.com/solid/node-solid-server/releases/tag/v5.3.0) 5.3.X and higher.
```

Comment:
Should we remove this altogether, or rephrase the prerequisite as "NSS versions prior 5.3.X are not supported"?

Reply:
remove please